app/code/Magento/ReviewGraphQl/Model/DataProvider/ReviewsDataProvider.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\ReviewGraphQl\Model\DataProvider;
+
+use Magento\Framework\GraphQl\Exception\GraphQlInputException;
+
+/**
+ * Provides reviews result
+ */
+class ReviewsDataProvider
```

Comment:
I would suggest using `AggregatedReviewsDataProvider` as a name. Just to make the class name more self-explanatory.